qt/widgets/instrumentview/inc/MantidQtWidgets/InstrumentView/UnwrappedSurface.h

```diff
@@ -92,9 +59,9 @@ class UnwrappedSurface : public ProjectionSurface {
 
   /** @name Implemented public virtual methods */
   //@{
-  void componentSelected(Mantid::Geometry::ComponentID = nullptr) override;
-  void getSelectedDetectors(QList<int> &dets) override;
-  void getMaskedDetectors(QList<int> &dets) const override;
+  void componentSelected(size_t componentIndex) override;
+  void getSelectedDetectors(std::vector<size_t> &dets) override;
```

Comment:
Would be useful to have a comment describing whether the container held ids or indexes. I expect it to be the latter. Consider renaming argument.